File: Benchmarks/enterprise/python/flask/testcode/benchmark_test_11594.py
# SPDX-License-Identifier: Apache-2.0
from flask import request, jsonify


def BenchmarkTest11594():
    xml_value = request.get_data(as_text=True)
    data = xml_value if xml_value else 'default'
    try:
        result = int(str(data))
    except Exception:
        pass
    return jsonify({"result": "success"})

File: Benchmarks/normal/java/jakarta/testcode/BenchmarkTest11594.java
// SPDX-License-Identifier: Apache-2.0
import jakarta.servlet.http.HttpServletRequest;
import jakarta.servlet.http.HttpServletResponse;
import jakarta.ws.rs.*;
import jakarta.ws.rs.core.Context;
import jakarta.ws.rs.core.MediaType;
import jakarta.ws.rs.core.Response;

@Path("/")
public class BenchmarkTest11594 {

    @GET
    @Path("/BenchmarkTest11594")
    @Produces(MediaType.APPLICATION_JSON)
    public Response BenchmarkTest11594(@HeaderParam("Host") String host, @Context HttpServletRequest request, @Context HttpServletResponse response) throws Exception {
        String hostValue = host != null ? host : "";
        String data = "" + hostValue;
        return Response.ok("<div>" + data + "</div>", MediaType.TEXT_HTML).build();
    }
}
